YoutubePlaylistDownloader/GlobalConsts.cs

```diff
@@ -403,6 +406,9 @@ public static async Task<string> TagFileBasedOnTitle(IVideo video, int vIndex, s
         }
         title = title.TrimStart(' ', '-', '[', ']');
 
+        if (returnTitleOlny)
```

Comment:
Please fix typo ("returnTitleOlny" to "returnTitleOnly"). And also in the method signature.